control/trajectory_follower/src/pid_longitudinal_controller_diagnostic.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Tier IV, Inc.
```

Comment:
Any reason to make another file? I feel that the class method implementation is better to be located in the same file.

I would be like
- 1. I want to see the diagnostic implementation in longitudinal controller.
- 2. I look into pid_longitudinal_controller.cpp, but not found
- 3. I got it, it must be implemented in hpp, and I look into pid_longitudinal_controller.hpp, but not found.
- 4. Then I grep, and find there is another file

Reply:
I'll fix it